CoreScriptsRoot/Modules/Server/ClientChat/DefaultClientChatModules/CommandModules/SwallowGuestChat.lua

```diff
@@ -1,34 +0,0 @@
---	// FileName: SwallowGuestChat.lua
---	// Written by: TheGamer101
---	// Description: Stop Guests from chatting and give them a message telling them to sign up.
```

Comment:
This module also shows a message that tells guests to sign up. This should be kept as guests might be confused why *they* cant chat. (because we're talking about children most of the time)

Reply:
See Core-Scripts/CoreScriptsRoot/Modules/Server/ClientChat/ChatBar.lua:
`function methods:DoLockChatBar()
	if self.TextLabel then
		if LocalPlayer.UserId > 0 then
			self.TextLabel.Text = ChatLocalization:Get(
				"GameChat_ChatMessageValidator_SettingsError",
				"To chat in game, turn on chat in your Privacy Settings."
			)
		else
			self.TextLabel.Text = ChatLocalization:Get(
				"GameChat_SwallowGuestChat_Message",
				"Sign up to chat in game."
			)
		end
		self:CalculateSize()
	end
	if self.TextBox then
		self.TextBox.Active = false
		self.TextBox.Focused:connect(function()
			self.TextBox:ReleaseFocus()
		end)
	end
end`

Reply:
Alright then, good to go.